keras/engine/sequential.py

```diff
@@ -0,0 +1,301 @@
+"""Sequential model class.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import warnings
+import copy
+
+from . import network
+from .training import Model
+from .base_layer import Layer
+from .input_layer import Input
+from .input_layer import InputLayer
+from .. import backend as K
+from .. import layers as layer_module
+
+try:
+    import h5py
+except ImportError:
+    h5py = None
```

Comment:
will this affect the sequential model if we dont have h5py?

Reply:
h5py is installed as part of keras-mxnet pypi

Reply:
Then why do we have this try catch?

Reply:
part of keras. did not want to update as it is an extra check and avoid extra conflict resolution.